<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cframe</class>
 <widget class="QMainWindow" name="cframe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1259</width>
    <height>836</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cframe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1241</width>
      <height>721</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>250</y>
        <width>211</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Cliente</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>360</y>
        <width>211</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>18</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Admin</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="btnReservar">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>210</y>
        <width>161</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Reservar Mesa</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMod">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>300</y>
        <width>211</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Modificar Reservacion</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCancel">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>390</y>
        <width>211</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Cancelar Reservacion</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLineEdit" name="nameLineEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="contactLineEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>45</y>
        <width>351</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>401</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Numero de Contacto SIN GUION:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>300</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Cantidad de Comensales:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="guestsSpinBox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>330</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>110</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>10</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2024</year>
        <month>8</month>
        <day>17</day>
       </datetime>
      </property>
     </widget>
     <widget class="QPushButton" name="createReservationButton">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>360</y>
        <width>211</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>14</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Crear Reservacion</string>
      </property>
     </widget>
     <widget class="QPushButton" name="viewAvailabilityButton">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>470</y>
        <width>211</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>14</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 170, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Ver disponiblidad</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>70</y>
        <width>351</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Fecha A Reservar:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>30</y>
        <width>351</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Fecha A Reservar:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="contactLineEdit2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>370</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Numero de referencia</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifyReservationButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>350</y>
        <width>271</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 85, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Modificar Reservacion</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>351</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Cantidad de Comensales:</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEdit2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>70</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>10</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2024</year>
        <month>8</month>
        <day>17</day>
       </datetime>
      </property>
     </widget>
     <widget class="QSpinBox" name="guestsSpinBox2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>310</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameLineEdit2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="referenceLineEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>400</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>401</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Numero de Contacto SIN GUION:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QPushButton" name="cancelReservationButton">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>430</y>
        <width>231</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>14</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Cancelar Reservacion</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>240</y>
        <width>441</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ingrese su Numero de referencia</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="referenceLineEdit2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>290</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6"/>
   </widget>
   <widget class="QPushButton" name="btnRegresar">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>740</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Regresar Menu Principal</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1259</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
